src/Sylius/Bundle/CoreBundle/Resources/config/app/workflow/sylius_catalog_promotion.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+use Sylius\Component\Core\Model\CatalogPromotion;
+use Symfony\Config\FrameworkConfig;
+
+return static function (FrameworkConfig $framework): void {
+    $catalogPromotion = $framework->workflows()->workflows('sylius_catalog_promotion');
+    $catalogPromotion
+        ->type('state_machine')
+        ->supports([CatalogPromotion::class])
+        ->initialMarking(['active']);
```

Comment:
The initial state should be inactive according to this fragment https://github.com/Sylius/Sylius/blob/1.13/src/Sylius/Component/Promotion/Model/CatalogPromotion.php#L43

What is more, we could use const for that :) 
```suggestion
        ->initialMarking([Sylius\Component\Promotion\Model\CatalogPromotionStates::STATE_INACTIVE]);
```